update to run the `pack` command (#703)
buildpacks/docs: content/docs/for-buildpack-authors/tutorials/basic-buildpack/02_building-blocks-cnb.md

```
<!--+- "{{execute}}"+-->

> Then run the following `pack` command after `cd` to `node-js-sample-app`:

Reviewer: ```suggestion
Then run the following `pack` command from the `node-js-sample-app` directory:
```
